src/Microsoft.VisualStudio.Services.Agent/HostContext.cs

```diff
@@ -134,7 +134,7 @@ public HostContext(string hostType, string logFile = null)
 
             // Enable Http trace
             bool enableHttpTrace;
-            if (bool.TryParse(Environment.GetEnvironmentVariable("VSTS_AGENT_HTTPTRACE"), out enableHttpTrace) && enableHttpTrace)
+            if (AgentKnobs.HttpTrace.GetValue(this).AsBoolean());
```

Comment:
I am guessing you did not mean to have that trailing semi-colon?

Reply:
Correct